cmd/root.go

```diff
@@ -279,6 +282,28 @@ func (r *Root) execute() {
 
 	//Set the version template for version command
 	rootCmd.SetVersionTemplate(`{{with .Version}}{{printf "Keploy %s" .}}{{end}}{{"\n"}}`)
+	currentVersion := utils.Version
+	// Show update message only if it's not a dev version
+	if !strings.HasSuffix(currentVersion, "-dev") {
+		// Check for the latest release version
+		releaseInfo, err := utils.GetLatestGitHubRelease()
+		if err != nil {
+			r.logger.Debug("Failed to fetch the latest release version", zap.Error(err))
+			return
```

Comment:
why are we returning if we failed to fetch the latest release, keploy should still work isn't it? This will confuse the user as to why keploy is not running